modules/3_detection/test/test_detector.cpp

```diff
@@ -68,17 +68,19 @@ TEST(detection, faces) {
     std::vector<Rect> boxes;
     std::vector<float> probs;
     std::vector<unsigned> classes;
-    model.detect(img, boxes, probs, classes);
+    model.detect(img, boxes, probs, classes,nmsThreshold, probThreshold);
 
     // Replace #if 0 to #if 1 for debug visualization.
-#if 0
-    for (int i = 0; i < boxes.size(); ++i) {
-        rectangle(img, boxes[i], 255);
-        putText(img, format("%.2f", probs[i]), Point(boxes[i].x, boxes[i].y - 2),
-                FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 255, 0));
-    }
+#if 1
+	for (int i = 0; i < boxes.size(); ++i) {
+		rectangle(img, boxes[i], Scalar(0, 0, 255));
+		rectangle(img, refBoxes[i], Scalar(0, 255, 0));
+		putText(img, format("%.2f", probs[i]), Point(boxes[i].x, boxes[i].y - 2),
+			FONT_HERSHEY_SIMPLEX, 0.5, Scalar(0, 255, 0));
+	}
     imshow("Detection", img);
```

Comment:
Please remove `imshow` to pass tests: https://travis-ci.org/github/dkurt/openvino_practice/builds/701245900
```
(Detection:5163): Gtk-WARNING **: cannot open display: 
```